docs/content/preview/drivers-orms/nodejs/sequelize-yugabytedb.md

```diff
@@ -31,42 +31,58 @@ showAsideToc: true
 
-## Working with Domain Objects
+## Working with domain objects
 
 This section describes how to use Node.js models (domain objects) to store and retrieve data from a YugabyteDB cluster.
 
+### Create a Node.js project and install Sequelize ORM core package
 
+Before proceeding with the next steps, you need to have Node.js installed on your machine. Refer to [Downloading and installing Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-installer-to-install-node-js-and-npm).
 
-### Creating a Node.js project and installing Sequelize ORM core package
+To create a simple Node.js project and install the `sequelize-yugabytedb` package, do the following:
 
-Before proceeding with the next steps, you need to have Node.js installed on your machine. To install it, refer to [Downloading and installing Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-installer-to-install-node-js-and-npm).
+1. Create a new directory.
 
-To create a simple Node.js project and install the `sequelize-yugabytedb` package, do the following:
+    ```sh
+    `mkdir nodejs-quickstart-example && cd nodejs-quickstart-example`
+    ```
 
-1. Create a new directory: 
-`mkdir nodejs-quickstart-example && cd nodejs-quickstart-example`
+1. Create a package.json file:
 
-2. Create a package.json file: 
-`echo {} > package.json`
+    ```sh
+    `echo {} > package.json`
+    ```
 
-3. Install sequelize-yugabytedb package: 
-`npm install sequelize-yugabytedb`
+1. Install sequelize-yugabytedb package:
 
-4. Create a empty demo.js file: 
-`touch example.js`
+    ```sh
+    `npm install sequelize-yugabytedb`
+    ```
 
-### Creating a Node.js example using Sequelize ORM
+1. Create an empty demo.js file:
 
-Add the following code in the `example.js` file. This code creates an Employees model to store and retrieve employee information. First it creates a connection by passing the basic connection parameters. Next, it defines the Employee model using the `define()` API, which specifies the type of information to store for an employee. The actual table is created by calling the `Employee.sync()` API in the `createTableAndInsert()` function. This also inserts the data for three employees into the table using the `Employee.create()` API. Finally, you can retrieve the information of all employees using `Employee.findAll()`.
+    ```sh
+    `touch example.js`
+    ```
 
-```js
+### Create a Node.js example using Sequelize ORM
+
+The following code creates an Employees model to store and retrieve employee information, as follows:
```

Comment:
Suggestion: remove "as follows" here (there's already "the following" in this sentence)